Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -730,6 +748,29 @@ public void ReadXmlBuildingJob(XmlReader reader)
     }
 
     /// <summary>
+    /// Reads the XML building job.
+    /// </summary>
+    /// <param name="reader">The XML reader to read from.</param>
+    public void ReadXmlDeconstructJob(XmlReader reader)
```

Comment:
Would it be better to create the job here as a Job prototype (similar to how is done with the build job) and then just clone it when deconstructing? If not. then why are we doing it for Build and not for Deconstruct, both should be similar.